applications/fax/src/fax_maintenance.erl

```diff
@@ -430,23 +429,3 @@ next_key(<<>>) ->
 next_key(Bin) ->
     <<Bin/binary, "\ufff0">>.
 
--spec load_smtp_attachment(kz_term:ne_binary(), kz_term:ne_binary()) -> 'ok'.
-load_smtp_attachment(DocId, Filename) ->
-    case file:read_file(Filename) of
-        {'ok', FileContents} ->
-            load_smtp_attachment(DocId, Filename, FileContents);
-        Error ->
-            io:format("error obtaining file ~s contents for docid ~s : ~p~n", [Filename, DocId, Error])
-    end.
-
--spec load_smtp_attachment(kz_term:ne_binary(), kz_term:ne_binary(), binary()) -> 'ok'.
-load_smtp_attachment(DocId, Filename, FileContents) ->
-    CT = kz_mime:from_filename(Filename),
-    case kz_datamgr:open_cache_doc(?KZ_FAXES_DB, DocId) of
-        {'ok', JObj} ->
-            case kzd_fax:save_outbound_fax(?KZ_FAXES_DB, JObj, FileContents, CT) of
-                {'ok', _Doc} -> io:format("attachment ~s for docid ~s recovered~n", [Filename, DocId]);
-                {'error', E} -> io:format("error attaching ~s to docid ~s : ~p~n", [Filename, DocId, E])
-            end;
-        {'error', E} -> io:format("error opening docid ~s for attaching ~s : ~p~n", [DocId, Filename, E])
-    end.
```

Comment:
@swysor why are we removing this ?

Reply:
It's not used anywhere.

Reply:
its a maintenance command. if smtp processing fails a system notify is sent and a admin can recover with this maintenance command

Reply:
Oh right, ok, I will restore that.

Reply:
the attachment is extracted from smtp and put into tmp dir. if it fails to upload to bigcouch a system alert is sent.
the fax doc is created but has no attachment, this allow a admin to retry saving the attachment to bigcouch.

Reply:
Fixed